src/OrchardCore/OrchardCore.Abstractions/Json/Serialization/DateTimeJsonConverter.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace OrchardCore.Json.Serialization;
+
+public class DateTimeJsonConverter : JsonConverter<DateTime>
+{
+    public override DateTime Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+    {
+        Debug.Assert(typeToConvert == typeof(DateTime));
+
+        if (!reader.TryGetDateTime(out DateTime value) && DateTime.TryParse(reader.GetString()!, out value))
+        {
+            return value;
+        }
+        
+        return value;
```

Comment:
```suggestion
        reader.TryGetDateTime(out DateTime value) && DateTime.TryParse(reader.GetString()!, out value);
        
        return value;
```

Reply:
Opps ! , we can't do that 。 The compilation failed. I restored it.